Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -77,28 +77,28 @@ public LayerMask[] MagneticSurfaces
 
         [SerializeField]
         [Tooltip("Max distance for raycast to check for surfaces")]
```

Comment:
Please using FormerlySerializedAs to avoid breaking previous users of this script